frontend/src/pages/ExploreData/ExploreDataPage.tsx

```diff
@@ -317,28 +322,24 @@ function ExploreDataPage(props: ExploreDataPageProps) {
         className={'h-full bg-exploreBgColor'}
       >
         <div
-          className={`z-almostTop mb-1 bg-white p-4 shadow-raised-tighter  transition-all duration-200 ease-in-out md:top-0 md:w-full
-            ${isSticking ? 'md:fixed' : ''}
+          ref={madlibRef}
+          className={`z-almostTop mb-1 bg-white p-4 shadow-raised-tighter md:top-0 md:w-full
+            ${!noTopicChosen ? 'md:sticky' : ''}
```

Comment:
We use `position: sticky` instead of `position: fixed` here to let browser handles the sticky behaviour for us.